install_config/persistent_storage/persistent_storage_csi.adoc

```diff
@@ -0,0 +1,358 @@
+[[install-config-persistent-storage-persistent-storage-csi]]
+= Persistent Storage Using Container Storage Interface (CSI)
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+Container Storage Interface (CSI) allows {product-title} to consume storage from storage backends that implement the link:https://github.com/container-storage-interface/spec[CSI interface] as xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage].
+
+
+[NOTE]
+====
+CSI volumes is a beta feature and may change in a future release of {product-title}. Due to its beta status, it does not support SELinux yet.
+====
+
+[NOTE]
+====
+{product-title} does not ship with any CSI drivers. It is recommended to use the CSI drivers provided by link:https://kubernetes-csi.github.io/docs/Drivers.html[community or storage vendors].
+====
+
+[NOTE]
+====
+{product-title} {product-version} supports version 0.2.0 of link:https://github.com/container-storage-interface/spec[CSI specification].
+====
+
+
+[[install-config-persistent-storage-csi-architecture]]
+== Architecture
+
+CSI drivers are typically shipped as container images. These containers are not aware of {product-title} where they run. To use CSI-compatible storage backed in {product-title}, the cluster administrator must deploy several components that serve as a bridge between {product-title} and the storage driver.
+
+The following diagram provides high-level overview about these components running in pods in {product-title} cluster.
```

Comment:
s/provides/provides a 

s/about these/about the

s/{product-title}/the {product-title}

Reply:
fixed